packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -0,0 +1,99 @@
+import { assetDataUtils, SignedOrder } from '@0x/order-utils';
+import { ERC20AssetData } from '@0x/types';
+import { BigNumber, logUtils } from '@0x/utils';
+import Axios, { AxiosResponse } from 'axios';
+
+import { MarketOperation } from '../types';
+
+/**
+ * Request quotes from RFQ-T providers
+ */
+export class QuoteRequestor {
+    private readonly _rfqtMakerEndpoints: string[];
+
+    constructor(rfqtMakerEndpoints: string[]) {
+        this._rfqtMakerEndpoints = rfqtMakerEndpoints;
+    }
+
+    public async requestRfqtFirmQuotesAsync(
+        makerAssetData: string,
+        takerAssetData: string,
+        assetFillAmount: BigNumber,
+        marketOperation: MarketOperation,
+        intentOnFilling: boolean,
```

Comment:
What is the point of this param? To get here this has to be `true`, and it never gets used in this function.

Reply:
It's not used yet because so far we're only supporting firm quotes.  This function will change such that if `intentOnFilling` is true, then it hits the maker's `/quote` endpoint, but if it's false then it will hit the maker's `/price` endpoint instead.

Reply:
I see. Will that be in this PR or soon after? I'm not usually a fan of placeholder code because sometimes we pivot and forget.

Reply:
Very soon after.  We're not putting off indicative quotes; they're a key part of RFQ-T.  I'm doing firm quotes in one milestone, and indicative in the next.

Reply:
I agree with @dorothy-zbornak, let's not add placeholder code (in the same that we don't leave commented out code).

Reply:
I removed this param for you guys, in https://github.com/0xProject/0x-monorepo/pull/2541/commits/227676c150a87404bce7e6d2e9b351ff2be137fa